site/content/en/latest/tasks/security/jwt-authentication.md

```diff
@@ -289,6 +292,113 @@ Additional connection settings for the remote JWKS host can be configured in the
+kind: SecurityPolicy
+metadata:
+  name: jwt-example
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: foo
+  jwt:
+    providers:
+    - name: example
+      localJWKS:
+        type: ValueRef
+        valueRef:
+          group: ""
+          kind: ConfigMap
+          name: jwt-local-jwks
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jwt-local-jwks
+data:
+  jwks: |
+    {
+        "keys": [
+            {
+                "alg": "RS256",
+                "e": "AQAB",
+                "key_ops": [
+                    "verify"
+                ],
+                "kty": "RSA",
+                "n": "xOHb-i1WDfeAvsbXTSOtosl3hCUDHQ8fRDqX_Rt998-hZDJmAoPOu4J-wcwq5aZtSn_iWUYLcK2WmC_1n-p1eyc-Pl4CBnxF7LUjCk-WGhniaCzXC5I5RON6c5N-MdE0UfukK0PM0zD3iQonZq0fIsnOYyFdYdWvQ5XW-C2aLlq2FUKrjmhAav10jIC0KGd2dHRzauzfLMUmt_iMnpU84Xrur1zRYzBO4D90rN0ypC2HH7o_zI8Osx4o1L8BScW78545sWyVbaprhBV1I2Sa4SH3NAc25ej3RIh-f13Yu97FVfO0AIG4VfFiaMmsTqNTCiBkM20tXD2Z-cHJTKemXzFgInJoqFLAkHLzJ0lPvAkKOgAOufLHa7RA-C276OXd72IXPsL1UOLN4sjhGqTtaynVa00yuHdi3f4-aoy9F9SUJeWfPg--nZNLzuI0eyufsTFywnx1bTQ_kdYlEr0dRE5sujlMk3cZ7FmOQRvcjA9MxFzoVKMmlZc6LMCgqw-P",
+                "use": "sig",
+                "kid": "b520b3c2c4bd75a10e9cebc9576933dc"
+            }
+        ]
+    }
+EOF
+```
+
+{{% /tab %}}
+{{% tab header="Apply from file" %}}
+Save and apply the following resource to your cluster:
+
+```yaml
+---
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: SecurityPolicy
+metadata:
+  name: jwt-example
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: foo
+  jwt:
+    providers:
+    - name: example
+      localJWKS:
+        type: ValueRef
+        valueRef:
+          group: ""
+          kind: ConfigMap
+          name: jwt-local-jwks
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jwt-local-jwks
+data:
+  jwks: |
+    {
+        "keys": [
+            {
+                "alg": "RS256",
+                "e": "AQAB",
+                "key_ops": [
+                    "verify"
+                ],
+                "kty": "RSA",
+                "n": "xOHb-i1WDfeAvsbXTSOtosl3hCUDHQ8fRDqX_Rt998-hZDJmAoPOu4J-wcwq5aZtSn_iWUYLcK2WmC_1n-p1eyc-Pl4CBnxF7LUjCk-WGhniaCzXC5I5RON6c5N-MdE0UfukK0PM0zD3iQonZq0fIsnOYyFdYdWvQ5XW-C2aLlq2FUKrjmhAav10jIC0KGd2dHRzauzfLMUmt_iMnpU84Xrur1zRYzBO4D90rN0ypC2HH7o_zI8Osx4o1L8BScW78545sWyVbaprhBV1I2Sa4SH3NAc25ej3RIh-f13Yu97FVfO0AIG4VfFiaMmsTqNTCiBkM20tXD2Z-cHJTKemXzFgInJoqFLAkHLzJ0lPvAkKOgAOufLHa7RA-C276OXd72IXPsL1UOLN4sjhGqTtaynVa00yuHdi3f4-aoy9F9SUJeWfPg--nZNLzuI0eyufsTFywnx1bTQ_kdYlEr0dRE5sujlMk3cZ7FmOQRvcjA9MxFzoVKMmlZc6LMCgqw-P",
+                "use": "sig",
+                "kid": "b520b3c2c4bd75a10e9cebc9576933dc"
+            }
+        ]
+    }
+```
+
+{{% /tab %}}
+{{< /tabpane >}}
+
+With the above configuration, the [SecurityPolicy] resource will use the JWKS stored in the `jwt-local-jwks` ConfigMap to authenticate requests for the `foo` HTTPRoute.
+
```

Comment:
would be better if you can add a example for `... to authenticate requests for the `foo` HTTPRoute`